v2/tools/generator/internal/astbuilder/comments_test.go

```diff
@@ -70,6 +71,8 @@ func TestWordWrap(t *testing.T) {
 		{"this is a simple line of text", 16, []string{"this is a simple ", "line of text"}},
 		{"this is a simple line of text", 20, []string{"this is a simple ", "line of text"}},
 		{"this is a simple line of text", 21, []string{"this is a simple line ", "of text"}},
+		{"", 0, []string{}},
+		{"this is a sample text", 0, []string{}},
```

Comment:
We should never throw away content like this - if someone inadvertently passes a width of `0` we want to return the content anyway. For consistency with existing behaviour when width is `1`, this should be:

```suggestion
		{"this is a sample text", 0, []string{"this ", "is ", "a ", "sample ", "text"}},
```